docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +106,255 @@ To manage file storages:
 
    ```
-   var newDir = documentsDir.createDirectory('newDir');
-   var anotherNewDir = documentsDir.createDirectory('newDir1/subNewDir1');
+   var newPath = 'documents/subDir'
+   var successCallback = function(newPath) {
+       console.log('New directory has been created: ' + newPath);
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.createDirectory('documents/newDir',successCallback, errorCallback);
    ```
 
-3. To delete a file, use the `deleteFile()` method:
+2. To delete a file, use the `deleteFile()` method:
 
    ```
-   function onDelete() {
-       console.log('deletedFile() is successfully done.');
+   var fullPath = 'documents/file_to_be_deleted.txt'
+   var callback = function(modifiedDirectory) {
+       console.log('deleteFile() is successfully done. Modified (parent) directory: ' + modifiedDirectory);
    }
-
-   documentsDir.deleteFile(newFile.fullPath, onDelete);
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.deleteFile(fullPath, callback, errorCallback);
    ```
 
-4. To delete a directory, use the `deleteDirectory()` method.
+3. To delete a directory, use the `deleteDirectory()` method.
 
    The second parameter defines whether the deletion is performed recursively for the sub-directories as well. If the parameter is set to `false`, the directory is deleted only if it is empty.
 
    ```
-   documentsDir.deleteDirectory(newDir.fullPath, false, onDelete);
-   anotherNewDir.parent.deleteDirectory(anotherNewDir.fullPath, false, onDelete);
+   var fullPath = 'documents/subDir'
+   var callback = function(modifiedDirectory) {
+       console.log('deleteDirectory() is successfully done. Modified (parent) directory: ' + modifiedDirectory);
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.deleteDirectory(fullPath, true, callback, errorCallback);
    ```
 
 ## Retrieving Files and File Details
 
-You can retrieve a list of files or file URIs using the `listFiles()` and `toURI()` methods. The URI can be used to identify the file, for example, by using it as the `src` attribute on an HTML `img` element.
+You can retrieve a list of files or file URIs using the `listDirectory()` and `toURI()` methods. The URI can be used to identify the file, for example, by using it as the `src` attribute on an HTML `img` element.
 
-You can retrieve file content as a `DOMString` with the `readAsText()` method. The encoding input parameter of the method defines the format in which the file content is returned.
+1. To retrieve a list of all the files and their directories located in a specified directory, use the `listDirectory()` method of the `filesystem` object. The method returns an array of `DOMString` elements, each beeing a path with name of file inside a directory pointed by `path`.
```

Comment:
To retrieve a list of all files and directories located in a ... - it is a general rule, so do not use 'the' statement